go.mod

```diff
@@ -1,6 +1,6 @@
 module k8s.io/cloud-provider-openstack
 
-go 1.19
+go 1.20
```

Comment:
https://github.com/kubernetes/kubernetes/blob/release-1.26/go.mod#L9 - this makes me uneasy about this patch. Any motivation to do that bump on our side, while K8s haven't?

Reply:
> Any motivation to do that bump on our side, while K8s haven't?
nope, I just synced it with the go compiler version.
usually this indicates that go needs to act in a different way even if the go binary is earlier than go.mod, but not in this case. IMHO we can skip this note . Just try to compile binaries with go 1.20.12 with go.mod 1.19 and then go.mod 1.20, you'll see that binaries are the same.

```sh
$ CGO_ENABLED=0 GOOS=linux GOARCH= GOPROXY=https://proxy.golang.org,direct go build -trimpath -o 1.20 cmd/openstack-cloud-controller-manager/main.go
$ sed -i 's/go 1.20/go 1.19/' go.mod
$ CGO_ENABLED=0 GOOS=linux GOARCH= GOPROXY=https://proxy.golang.org,direct go build -trimpath -o 1.19 cmd/openstack-cloud-controller-manager/main.go
$ diff -u <(xxd 1.19) <(xxd 1.20) | colordiff
```

Reply:
Okay, I think I'd rather keep 1.19 there then.

Reply:
done